### PR TITLE
chg: atis freqs

### PR DESCRIPTION
**Pull request by 132nd-etcher**
Change  ATIS fequencies from XXX.400 to XXX.300 (for all airfields).

Requested by @132ndNeck

### CODE REVIEW

File: esst/atis/_atis_airfields.py [hunk 1]
```diff
@@ -7,107 +7,107 @@
...
 URKG = Airfield('URKG', 'Geledzhik',
                 URCoord('44.592300', '38.023300', '82'),
-                URFrequency('126.400;A'),
+                URFrequency('126.300;A'),
                 '04,22')
 # noinspection SpellCheckingInspection
 URKH = Airfield('URKH', 'Maykop - Khanskaya',
-                URCoord('44.681400', '40.032501', '590'),
-                URFrequency('125.400;A'),
+                URCoord('44.68.300', '40.032501', '590'),
```
Comment (132nd-etcher) [inline esst/atis/_atis_airfields.py]: This is a mistake ...

**Comment by 132ndNeck** [comment]
I am not sure what to do when requested for a review? Should I check anything, and in that case  what?
I am happy and want to change ATIS freq from XXX.4 to XXX.3 for all airfields.

**Comment by 132nd-etcher** [comment]
@132ndNeck I used this to "ping" people. In general, when requested for a review, you would go over the changes in the code-base, and indicate if you are happy with them or not. It's a simple way to discuss atomic changes and comment over lines or blocks of line. It's also an iterative process, which helps compartmentalising the discussions over change sets.

**Comment by 132ndNeck** [comment]
Sounds good, its what I assumed. I do not know anything about the code, so I can be of much value for that. But always happy with come with necessary feedback or inputs.

**Pull request merged**